src/main/kotlin/lotto/LottoPrinter.kt

```diff
@@ -0,0 +1,16 @@
+package lotto
+
+class LottoPrinter(val amountOfMoney: Int) {
+    val amountOfTicket: Int = amountOfMoney / LottoTicket.COST_OF_TICKET
+    val bundleOfLottoTicket: List<LottoTicket> = generateLottoTickets(amountOfTicket)
+
+    fun generateLottoTickets(amountOfTicket: Int): List<LottoTicket> {
+        val tempList = mutableListOf<LottoTicket>()
+        (1..amountOfTicket).forEach {
+            it
+            val ticket = LottoTicket(RandomNumberGenerator.generateNumber())
+            tempList.add(ticket)
+        }
```

Comment:
```suggestion
        repeat((1..amountOfTicket).count()) {
            val ticket = LottoTicket(RandomNumberGenerator.generateNumber())
            tempList.add(ticket)
        }
```

You could express this logic in another way by using `repeat()`.

Reply:
Thank you for the suggestion! `repeat()` seems very useful! Thanks!